GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -144,12 +169,33 @@ function CreateMetaInstaller()
         exit 1
     fi
     
-    METAPACKAGENAME="$INSTALLERPACKAGENAME-Git.$GITVERSION.pkg"
+    copyGitPkgCmd="/bin/cp -Rf \"${GITINSTALLERPATH}\" \"${PACKAGESTAGINGDIR}/.\""
+    eval $copyGitPkgCmd
     
-    buildMetapkgCmd="/usr/bin/productbuild --package \"$GITINSTALLERPATH\" --package \"${BUILDOUTPUTDIR}\"$INSTALLERPACKAGENAME.pkg \"${BUILDOUTPUTDIR}\"$METAPACKAGENAME"
+    GITPKGNAME="$GITINSTALLERNAME.pkg"
+    GIT_CHOICE_OUTLINE_ELEMENT_TEXT="<line choice=\"com.git.pkg\"/>"
+	GIT_CHOICE_ID_ELEMENT_TEXT="<choice id=\"com.git.pkg\" visible=\"false\"> <pkg-ref id=\"com.git.pkg\"/> </choice>"
+	GIT_PKG_REF_ELEMENT_TEXT="<pkg-ref id=\"com.git.pkg\" version=\"$GITVERSIONSTRING\" onConclusion=\"none\">$GITPKGNAME</pkg-ref>"
+
+	# 1.  Update Distribution.xml
+	# 1.1 Substitute Git placeholders with actual values
+    /usr/bin/sed -e "s|GIT_CHOICE_OUTLINE_PLACEHOLDER|$GIT_CHOICE_OUTLINE_ELEMENT_TEXT|g" "$SCRIPTSPATH/Distribution.xml" > "${BUILDOUTPUTDIR}/Distribution.updated.xml"
+	/usr/bin/sed -i.bak "s|GIT_CHOICE_ID_PLACEHOLDER|$GIT_CHOICE_ID_ELEMENT_TEXT|g" "${BUILDOUTPUTDIR}/Distribution.updated.xml"
+	/usr/bin/sed -i.bak "s|GIT_PKG_REF_PLACEHOLDER|$GIT_PKG_REF_ELEMENT_TEXT|g" "${BUILDOUTPUTDIR}/Distribution.updated.xml"
+	
+	# 1.2 Substitute VFSForGit placeholders with actual values
+	/usr/bin/sed -i.bak "s|VFSFORGIT_VERSION_PLACHOLDER|$PACKAGEVERSION|g" "${BUILDOUTPUTDIR}/Distribution.updated.xml"
+	/usr/bin/sed -i.bak "s|VFSFORGIT_PKG_NAME_PLACEHOLDER|$INSTALLERPACKAGENAME.pkg|g" "${BUILDOUTPUTDIR}/Distribution.updated.xml"
```

Comment:
Generating templated XML this way seems like it's taking shell scripting a bit too far if you ask me, but I guess I'm not the one maintaining this script. 🙃

Reply:
Do you have a better suggestion for handling the version numbers here? I also am a little scared of this, but I don't have any other ideas off the top of my head.

Reply:
As far as I'm aware, `xsltproc` ships with macOS and can be used for some basic XML processing. There's also scripting languages like python, ruby, and perl where the xml library packages are probably also included.

Or maybe put together a very small .NET program to generate the XML? That would probably fit best with the rest of the project so far… (Is JScript still a thing?)

Reply:
Have removed some duplicate code - it is slightly easier to follow now.

Reply:
This is much smaller now so I'm happy with this relatively cheap solution versus introducing another scripting language or writing a mini .NET program. If this gets more complicated, I think that's probably the way we should go though.